const loginUser = () => {
    try {
        
    } catch (error) {
        console.log(error)
    }
}
const registerUser = async (req,res) => {
    try {
        

    } catch (error) {
        console.log(error)
    }
}